table/meta_blocks.h

```diff
@@ -71,8 +71,10 @@ class PropertyBlockBuilder {
   Slice Finish();
 
  private:
+  const Comparator* comparator_ = BytewiseComparator();
```

Comment:
Is this only for assertion? If yes, may worth guarding under dbg macro and have a more specific name.